src/iracingdataapi/client.py

```diff
@@ -64,7 +64,7 @@ def _get_resource_or_link(self, url, payload=None):
             print("Rate limited, waiting")
             reset_datetime = datetime.fromtimestamp(int(r.headers["x-ratelimit-reset"]))
             delta = reset_datetime - datetime.now()
-            if delta > 0:
+            if delta.total_seconds() > 0:
                 time.sleep(delta.total_seconds())
```

Comment:
Maybe avoid to recalculate seconds, and store it in a variable?

```suggestion
            seconds_to_unlock = delta.total_seconds()
            if seconds_to_unlock > 0:
                time.sleep(seconds_to_unlock)
```

Reply:
Since we only do this twice and this part of the code will rarely be called, I don't think this would make a huge difference.